packages/faceted-search/src/components/FacetedToolbar/FacetedToolbar.module.scss

```diff
@@ -11,13 +12,13 @@ $toolbar-height: 5.5rem;
 	position: relative;
 	width: 100%;
 
-	.tc-faceted-switch-mode{
+	.tc-faceted-switch-mode {
 		width: auto;
 		margin-right: $padding-small;
 	}
 
 	&-buttons {
```

Comment:
Not even sure it is used for all the `&-buttons {}` block

Reply:
Yes but it seems it accept children so hard to predict. I would let it as it is for now